root/etc/httpd/conf.d/janus-admin-api.conf

```diff
@@ -0,0 +1,5 @@
+Alias "/janus-admin-api" "/opt/janus/share/janus/demos"
+<Directory "/opt/janus/share/janus/demos/">
+    Options FollowSymLinks
+    Require all granted
+</Directory>
```

Comment:
What is this fragment? Why is it needed?

Isn't already possible to talk with the admin API on  http://127.0.0.1:7088?

Reply:
I was about to ask the same thing :D

Reply:
Maybe it's worth to add some examples of `curl` calls that interacts with the administrative API...

Reply:
You are right ! :)
Once you have enabled the admin api, you can interact with them on `/admin` path.
Janus provides some demo pages on the above path.
Included into the demo pages, there is the admin.html page, that is an already developed ready to use page that provides some useful information (using admin apis), such as:
- janus version
- all the sessions with the associated plugin handles (useful to debug problems on ice,ssl,streams, etc...)
- you change the log level at runtime
- ...

You can take a look at the https://janus.conf.meetecho.com/admin.html
Without the admin.html page, you have to develop something similar.

Security aspects:
once you have open admin.html, you have to specify the admin url and the admin secret

Reply:
I'm afraid it is quite complex to use that endpoint: it would require to port forward both 443 and 7088 TCP ports to use it?

Can we just document some `curl` calls? or write a little helper script for that?

Reply:
If nethvoice is reachable from outside, you have only to create a tunnel for 7088, otherwise you are right, you have to port forward also 443, but in this case I think it is necessary even if you don't need to debug janus.
I think also that the port forward is needed also with the usage of curl, with the difference that you have to create a script to check something: for example to have all the plugin sip handles of a specific session, you have to do 2 curl requests, instead of open the web page admin.html.

I think admin.html offers something ready to use to simplify some basic usage, but this is only my opinion: if you think that develop some additional scripts is a better way for me is ok.

Reply:
- `curl` can be invoked on the local system: no port forwards are needed.
- If the ssh tunnel is just for port 7088, are we going to hit Same-Origin policy restrictions?

Reply:
- No Same-Origin policy, example:
![image](https://user-images.githubusercontent.com/1625334/86242801-88f55980-bba5-11ea-978d-c5d6c5a7b26c.png)